src/components/LandingPage/index.tsx

```diff
@@ -44,15 +44,19 @@ const LandingPage = (): JSX.Element => {
                             </p>
                         </div>
                     </Link>
-                    <div className={styles.borderedContent}>
+
+                    <Link
+                        to={ABOUT_PATHNAME}
+                        className={styles.borderedContent}
```

Comment:
```suggestion
                        className={styles.borderedContent}
                        to={ABOUT_PATHNAME}
```
Teeny nit: reverse to match above